lib/parser/lexer.rl

```diff
@@ -87,6 +87,11 @@ class Parser::Lexer
     'v' => "\v", '\\' => "\\"
   }
 
+  BLANK_STRING = ''.freeze
+  ESCAPED_NEXT_LINE = "\\\n".freeze
+  REGEXP_META_CHARACTERS = "\\$()*+.<>?[]^{|}".freeze
```

Comment:
`REGEXP_META_CHARACTERS = Regexp.union(*"your_chars_string".chars)` should do the trick.